apps/wallet-mobile/src/WalletInit/MnemonicCheck/MnemonicCheckScreen.tsx

```diff
@@ -70,7 +73,9 @@ export const MnemonicCheckScreen = () => {
       <View style={styles.buttons}>
         <Button
           block
-          onPress={() => createWallet({name, mnemonicPhrase: mnemonic, password, networkId, walletImplementationId})}
+          onPress={() =>
```

Comment:
Maybe it would be interesting to move this outside. Not important, nitpicking issue

Reply:
agreed is bloating the component